<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleName">
   <string>Главная</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="printsettingsbutton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>420</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Print settings</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>0</y>
      <width>20</width>
      <height>491</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="bot_control_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Запустить телеграмм бота</string>
     </property>
    </widget>
    <widget class="QLabel" name="botStatusLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>БОТ:</string>
     </property>
    </widget>
    <widget class="QLabel" name="botStatusLabel_ind">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>Остановлен</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="serverGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Server</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ядро :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Версия :</string>
     </property>
    </widget>
    <widget class="QLabel" name="coreLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="versionLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Запустить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Остановить</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>56</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Игроки онлайн :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Времени онлайн :</string>
     </property>
    </widget>
    <widget class="QLabel" name="versionLabel_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="versionLabel_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Настроить</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="coreSelectBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>140</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="upload_core_action"/>
    <addaction name="auto_start_last_core_action"/>
    <addaction name="core_select_action"/>
    <addaction name="separator"/>
    <addaction name="open_setings_action"/>
    <addaction name="separator"/>
    <addaction name="restart_action"/>
    <addaction name="to_trey_action"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_GitHub"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Свернуть в трей</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="action_GitHub">
   <property name="text">
    <string>Репозиторий GitHub</string>
   </property>
  </action>
  <action name="upload_core_action">
   <property name="text">
    <string>Добавить ядро</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Запутить ядро(выбрать)</string>
   </property>
  </action>
  <action name="auto_start_last_core_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Запомнить последний профиль</string>
   </property>
  </action>
  <action name="core_select_action">
   <property name="text">
    <string>Запутить ядро(выбрать)</string>
   </property>
  </action>
  <action name="open_setings_action">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Свернуть в трей</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Закрыть программу</string>
   </property>
  </action>
  <action name="restart_action">
   <property name="text">
    <string>Перезапустить программу</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Закрыть программу</string>
   </property>
  </action>
  <action name="to_trey_action">
   <property name="text">
    <string>Свернуть в трей</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Закрыть программу</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
